Diary/Diary/DiaryService.cs

```diff
@@ -0,0 +1,220 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Diary
+{
+    internal class DiaryService
+    {
+        static readonly PriorityTask[] weeklyTasks = new PriorityTask[100];
+        internal static int RecodrTask(int numberTask)
```

Comment:
* empty line
* missing access modifier on `weeklyTasks`
* you use the highest type `PriorityTask` to store tasks?
* RecodrTask -> RecordTask (install spell checker)
* bad design - `numberTask` should not be returned, if you need an actual count create property
```c#
public int Count => weeklyTasks.Length;
```